sdk/RELEASE.md

```diff
@@ -1,19 +1,31 @@
-# Current Version (Still in Development)
+# Current Version (in development)
 
-## Major Features and Improvements
+## Features
 
-## Breaking Changes
+## Breaking changes
 
-### For Pipeline Authors
+## Deprecations
+
+## Bug fixes and small changes
 
-### For Component Authors
+
+## Documentation updates
+
+# 2.0.0-beta.5
+
+## Features
+* Support `google.`-namespaced artifact types [\#8191](https://github.com/kubeflow/pipelines/pull/8191), [\#8232](https://github.com/kubeflow/pipelines/pull/8232), [\#8233](https://github.com/kubeflow/pipelines/pull/8233), [\#8279](https://github.com/kubeflow/pipelines/pull/8279)
+* Support dynamic importer metadata [\#7660](https://github.com/kubeflow/pipelines/pull/7660)
+
+## Breaking changes
 
 ## Deprecations
 
-## Bug Fixes and Other Changes
```

Comment:
I would not replace "other" with "small" here. There could be large refactoring changes for example which are not "small" yet doesn't fit "Major features and improvements" or "Features"

Reply:
Agreed. Will revert this.